packages/react-components/source/react/library/tag/Tag.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { node, func } from 'prop-types';
+import Button from '../button';
+import Text from '../text';
+
+const propTypes = {
+  /** Tag text or other content */
+  children: node.isRequired,
```

Comment:
Unless we know we need to accept arbitrary content, can we add something like a `label` string prop instead? (I could see the future use of an icon on the left, but we can either add a new prop for that when the time comes or readdress children.) To be clear, I'm usually in favor of adding the "escape hatch" of adding `children` as a prop to increase flexibility, but in this case, we're going to need to style the text and a separate prop would leave the door open for adding `aria-labelledby`.